Header header
uint8 state
uint8 OPEN=0
uint8 CLOSE=1
